css/_welcome_page.scss

```diff
@@ -43,6 +43,10 @@ body.welcome-page {
             line-height: $welcomePageHeaderTextTitleLineHeight;
             margin-bottom: $welcomePageHeaderTextTitleMarginBottom;
             opacity: $welcomePageHeaderTextTitleOpacity;
+
+            // Fix #4814: refer _base.scss
```

Comment:
Personally, I do not see, how mentioning the issue is useful in a comment? That’s what the git log is for, isn’t it? I’d prefer a summary, if a comment is warranted.

Reply:
I don't have a special preference for that, while I strongly believe leaving a comment on CSS / SCSS / LESS helps to avoid a visual regression, as long as styles are maintained on different files from components. If rewriting a comment to `Cancel user-select: none on _base.scss` etc. would make better sense, I would do so.